Project02Eclipse/src/com/hemebiotech/analytics/ProcessOut.java

```diff
@@ -0,0 +1,42 @@
+package com.hemebiotech.analytics;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
+public class ProcessOut {
+
+    /**
+     * Properties
+     */
+    private ISymptomReader reader;
+    private FileWriter writer;
+
+    /**
+     * Constructor
+     *
+     * @param reader refer to object instance of ReadSymptomDataFromFile
+     * @param writer refer to object instance of output file
+     */
+    public ProcessOut(ISymptomReader reader, FileWriter writer) {
+        this.reader = reader;
+        this.writer = writer;
+    }
+
+    /**
+     * Transform the list to a HashSet to keep just one copy of each symptoms
+     * Transform it to list for sort alphabetically
+     *
+     * @for each element of the list : write in output file his name and his occurrences
+     */
+    public void ProcessOut() throws IOException {
+        Set<String> test = new HashSet<>(reader.GetSymptoms());
```

Comment:
what is a `test`? It could be renamed as `symptoms` what do you think?

Reply:
Yep its better, it was a mistake